classes/PodsAdmin.php

```diff
@@ -2090,7 +2090,7 @@ public function admin_components () {
             if ( !empty( $component_data[ 'category' ] ) ) {
                 $category_url = pods_query_arg( array( 'view' => sanitize_title( $component_data[ 'category' ] ), 'pg' => '', 'page' => $_GET[ 'page' ] ) );
 
-                $component_data[ 'category' ] = '<a href="' . esc_url( $category_url ) . '">' . $component_data[ 'category' ] . '</a>';
+                $component_data[ 'category' ] = '<a href="' . esc_url( $category_url ) . '">' . __( $component_data[ 'category' ], 'pods' ) . '</a>';
```

Comment:
I agree completely.
I'll revert this part of the PR and hope that the `Category` will be localized in its origin in the future, because having the top categories translated and the categories in the column in english is a bit odd.
As the metadata is in the component header in a comment, I can't make the gettext function to work.
I believe it would be better for localization if all that metadata is generated without a commented header but in the code below the header.